website/src/app/api/.well-known/nostr.json/route.ts

```diff
@@ -0,0 +1,31 @@
+import { NextResponse, NextRequest } from 'next/server';
+
+const nostr_keys = {
+  joyboy: {
+    pubkey: 'npub1nq5ckz62p4vxwu08lpx8ggu5k5qn6d7pdtcfyj7hae3wc6j30fwseex2rq',
+  },
+  testnet: {
+    pubkey: 'b0635d6a9851d3aed0cd6c495b282167acf761729078d975fc341b22650b07b9'
+  }
+};
+
+export async function GET(request: NextRequest) {
```

Comment:
Will this be available at `/.well-known/nostr.json` or `/api/.well-known/nostr.json`?

Reply:
oh yeah it's suppose to be available at ```/.well-known/nostr.json```. I'll refactor it